src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            String data = readFile(fromFileName);
+            String report = generateReport(data);
+            writeToFile(toFileName, report);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private String generateReport(String data) {
+        StringBuilder report = new StringBuilder();
+        data = data.replaceAll("\\[|\\]", "");
+        String[] stringArray = data.split(", ");
+        System.out.println(data);
+        report.append("supply,").append(Integer.parseInt(stringArray[0]))
+                .append(System.lineSeparator());
+        report.append("buy,").append(Integer.parseInt(stringArray[1]))
+                .append(System.lineSeparator());
+        report.append("result,").append(Integer.parseInt(stringArray[2]))
+                .append(System.lineSeparator());
+        return report.toString();
+    }
+
+    private String readFile(String fileName) throws IOException {
```

Comment:
It is redundant, you cached all exceptions here, 
```suggestion
    private String readFile(String fileName){
```